src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
+@import './variebles';
+@import './mixins';
+
 body {
```

Comment:
The body tag should not be styled directly. Better to add a class for the body and apply styles to it.